src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="sb-autocomp-dropdown">
   <mat-form-field class="sb-autocomp-dropdown__formfield">
-    <input matInput (click)="DisplayDropdown()" readonly name="selectedDpdwnInput"
+    <input matInput (focus)="DisplayDropdown()" readonly name="selectedDpdwnInput"
       [value]="_selectedDpdwnInput ? _selectedDpdwnInput : ''" />
   </mat-form-field>
   <div class="sb-autocomp-dropdown__list" *ngIf="displayDropdown">
-    <input matInput #autocompleteInput placeholder="Search" autofocus />
+    <input matInput #autocompleteInput placeholder="Search" name="search"/>
```

Comment:
why is the name removed?

Reply:
auto focus not working, so written code in .ts to make it work.

Reply:
so removed from HtmlTemplate